src/internationalization/dictionaries/pt.json

```diff
@@ -48,6 +48,21 @@
     "projects": "Projetos"
   },
   "about": {
+    "sections": {
+      "cesium": {
+        "title": "O que é o CeSIUM",
+        "subtitle": "O CeSIUM é um Centro de estudantes de Engenharia Informática da Universidade do Minho composto por alunos voluntários que têm como objectivo principal representar e promover o curso. É um núcleo unido e dinâmico capaz de proporcionar experiências únicas e enriquecedoras para a tua futura vida profissional.",
+        "description": "Procuramos organizar actividades de contéudo semelhante e complementar ao curso, assim como promover e divulgar acções de lazer e socialização entre estudantes e ex-alunos. Representamos ainda os interesses colectivos dos alunos de Engenharia Informática junto dos orgãos ou instituições para que a sua carreira académica seja mais rica."
+      },
+      "team": {
+        "title": "Team",
+        "subtitle": "O CeSIUM não existiria sem a nossa equipa incrível, que trabalha todos os dias para manter o núcleo a funcionar e tornar os anos dos estudantes no curso únicos. Conhece-a aqui."
+      },
```

Comment:
Since this PR is only about the Departments section, lets not input this information yet and leave it for the PRs that tackle these, just to be more organized ;) (remove this also on the english dict)

```suggestion
```